### ISSUE TITLE
Yolo v4 Train error

### ISSUE THREAD

**Issue by igoriok1994** [AlexeyAB/darknet#6867]
Hi,

I have Core i7 64GB RAM, Quadro M1000M 2GB VRAM.

I'm getting error when try to train custom object detector.

Parameters:

```
batch=64
subdivisions=16
width=416
height=416
optimized_memory=3
workspace_size_limit_MB=500
```

I have `5 classes`, so `filters` before `[yolo] layers` = 30

Versions:

![image](https://user-images.githubusercontent.com/37294916/96910625-7383a280-14a8-11eb-8c49-219f0dc22ea3.png)

Error:

![image](https://user-images.githubusercontent.com/37294916/96910646-7a121a00-14a8-11eb-9e68-42d52e7022e6.png)

Command:

```
darknet.exe detector train cfg/afm.data cfg/yolov4-custom.cfg cfg/yolov4.conv.137 -dont-show
```

**Comment by adhilcodes** [comment]
**Change makefile to have GPU and OPENCV enabled**

```
%cd darknet
!sed -i 's/OPENCV=0/OPENCV=1/' Makefile
!sed -i 's/GPU=0/GPU=1/' Makefile
!sed -i 's/CUDNN=0/CUDNN=1/' Makefile
!sed -i 's/CUDNN_HALF=0/CUDNN_HALF=1/' Makefile
```
**# Verify CUDA**

`!/usr/local/cuda/bin/nvcc --version`

verify by running the above code before training . Also check that your train.txt & test.txt file are generated .

**Comment by igoriok1994** [comment]
> **Change makefile to have GPU and OPENCV enabled**
> **# Verify CUDA**

Code above can be run with `cmd`?

**Comment by adhilcodes** [comment]
@igoriok1994  Yes you can, Remember to **remove** the **percentage(%) & exclamation(!)** symbols if you are running it in `cmd `

**Comment by igoriok1994** [comment]
Strange,. but via I get error via console that `sed` is not a command or package file.

**Comment by stephanecharette** [comment]
Not strange at all, `sed` is a Linux command unless you find and install a Windows port.

The Windows version of darknet doesn't use the Makefile anyway.  I would follow the instructions on building Darknet on Windows.

For more information, you can take a look at the "#building-in-windows" channel on the Darknet/YOLO discord:  https://discord.gg/zSq8rtW

**Comment by igoriok1994** [comment]
Okay. 

I was followed these instructions to install Yolov4 (https://www.youtube.com/watch?v=5pYh1rFnNZs)
Detection with pre-trained weights works perfectly, but when I try to train my own object detector (with my data) I get error above.

**Comment by snehsagarajput** [comment]
> Strange, but via `CMD` I get error that `sed` is not a command or package file.

If you cannot use the sed command simply open the darknet directory then open the Makefile(in the darknet directory) in any of the text editor and simply make the following changes  in the first 4 lines only :

OPENCV=0  ->  OPENCV=1
GPU=0         ->  GPU=1
CUDNN=0   ->  CUDNN=1
CUDNN_HALF=0 ->  CUDNN_HALF=1
